src/pipelines/master_pipeline.py

```diff
@@ -292,6 +303,14 @@ def get_args(
     if len(features) == 0:
```

Comment:
You're right. If someone only selected W2V_MIN and W2V_PRETRAINED, the w2v dict would not be built with the correct PRETRAINED value because the if clause above the w2v dict creation didn't check for MIN/MAX/ABS being set. I have changed it. I realize now that Sacred won't see the change (when we set W2V_AVG or W2V_APPEND if the user didn't select both types) but we will know what happened.